src/ontology/mondo-edit.obo

```diff
@@ -430439,6 +430582,410 @@ property_value: IAO:0000231 "out of scope" xsd:string
+xref: Orphanet:580951 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005328 {source="https://orcid.org/0000-0001-5208-3432"} ! eye disorder
+
+[Term]
+id: MONDO:0035592
+name: congenital infiltrating lipomatosis of the face
+subset: ordo_disorder
+synonym: "CIL-F" EXACT [Orphanet:583097]
+synonym: "facial infused lipomatosis" EXACT [Orphanet:583097]
+synonym: "fibroadipose infiltrating lipomatosis" EXACT [Orphanet:583097]
+xref: ICD10:Q87.3 {source="Orphanet:583097", source="MONDO:mondoSubClassOfSource"}
+xref: Orphanet:583097 {source="MONDO:equivalentTo"}
+is_a: MONDO:0006574 {source="https://orcid.org/0000-0001-5208-3432"} ! lipomatosis
+
+[Term]
+id: MONDO:0035614
+name: sporadic fatal insomnia
+def: "A rare sporadic human prion disease characterized by adult onset of progredient neurodegeneration presenting as a combination of psychiatric, sleep, and oculomotor disturbances, with development of progressive cognitive impairment (the predominantly affected cognitive domains being memory, temporal and/or spatial orientation, language, executive functions, and attention), postural instability, and sometimes additional motor abnormalities and autonomic hyperactivity, in the course of the disease. Bilateral thalamic hypometabolism on FDG-PET imaging and positive prion seeding activity in the cerebrospinal fluid are present in many cases. The disease is fatal within typically two to three years." [Orphanet:586130]
+subset: ordo_disorder
+xref: ICD10:A81.9 {source="MONDO:mondoSubClassOfSource", source="Orphanet:586130"}
+xref: Orphanet:586130 {source="MONDO:equivalentTo"}
+is_a: MONDO:0018926 {source="https://orcid.org/0000-0001-5208-3432"} ! human prion disease
+relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/9449 ! PRNP
+
+[Term]
+id: MONDO:0035645
+name: inherited gynecological cancer-predisposing syndrome
+subset: ordo_group_of_disorders
+xref: Orphanet:589746 {source="MONDO:equivalentTo"}
+is_a: MONDO:0015356 {source="Orphanet:589746"} ! hereditary neoplastic syndrome
+
+[Term]
+id: MONDO:0035684
+name: epidermolysis bullosa simplex without extracutaneous involvement
+subset: ordo_group_of_disorders
+synonym: "EBS without extracutaneous involvement" EXACT [Orphanet:595346]
+xref: Orphanet:595346 {source="MONDO:equivalentTo"}
+is_a: MONDO:0017610 {source="Orphanet:595346"} ! epidermolysis bullosa simplex
+
+[Term]
+id: MONDO:0035685
+name: epidermolysis bullosa simplex with extracutaneous involvement
+subset: ordo_group_of_disorders
+synonym: "EBS with extracutaneous involvement" EXACT [Orphanet:595351]
+xref: Orphanet:595351 {source="MONDO:equivalentTo"}
+is_a: MONDO:0017610 {source="Orphanet:595351"} ! epidermolysis bullosa simplex
+
+[Term]
+id: MONDO:0035689
+name: syndrome of reduced sensitivity to thyroid hormone
+subset: ordo_group_of_disorders
+xref: Orphanet:596426 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003240 {source="https://orcid.org/0000-0001-5208-3432"} ! thyroid gland disorder
```

Comment:
I don't think this is correct. The thyroid gland is the one producing the thyroid hormone, BUT sensitivity to this hormone could be due to anything else than the thyroid gland which could be completely fine and functional. 
I would say this should be "syndromic disease"

Reply:
fixed